examples/YOLOv8-SAHI-Inference-Video/yolov8_sahi.py

```diff
@@ -6,32 +6,32 @@
 import cv2
 from sahi import AutoDetectionModel
 from sahi.predict import get_sliced_prediction
-from sahi.utils.yolov8 import download_yolov8s_model
+from sahi.utils.ultralytics import download_yolo11n_model
 
 from ultralytics.utils.files import increment_path
 from ultralytics.utils.plotting import Annotator, colors
 
 
 class SAHIInference:
-    """Runs YOLOv8 and SAHI for object detection on video with options to view, save, and track results."""
+    """Runs Ultralytics YOLO11 and SAHI for object detection on video with options to view, save, and track results."""
 
     def __init__(self):
-        """Initializes the SAHIInference class for performing sliced inference using SAHI with YOLOv8 models."""
+        """Initializes the SAHIInference class for performing sliced inference using SAHI with YOLO11 models."""
         self.detection_model = None
 
     def load_model(self, weights):
-        """Loads a YOLOv8 model with specified weights for object detection using SAHI."""
-        yolov8_model_path = f"models/{weights}"
-        download_yolov8s_model(yolov8_model_path)
+        """Loads a YOLO11 model with specified weights for object detection using SAHI."""
+        yolo11_model_path = f"models/{weights}"
+        download_yolo11n_model(yolo11_model_path)
         self.detection_model = AutoDetectionModel.from_pretrained(
-            model_type="yolov8", model_path=yolov8_model_path, confidence_threshold=0.3, device="cpu"
+            model_type="yolo11", model_path=yolo11_model_path, confidence_threshold=0.3, device="cpu"
```

Comment:
up to you @RizwanMunawar 

```suggestion
            model_type="ultralytics", model_path=yolo11_model_path, confidence_threshold=0.3, device="cpu"
```